velox/type/Timestamp.h

```diff
@@ -316,7 +316,7 @@ struct Timestamp {
   // Example: Timestamp ts{0, 0};
   // ts.Timezone("America/Los_Angeles");
   // ts.toString() returns December 31, 1969 16:00:00
-  void toTimezone(const date::time_zone& zone);
+  void toTimezone(const date::time_zone& zone, bool allowOverflow = false);
```

Comment:
nit: better to document the behaviors when allowOverflow if true/false.

Reply:
Thanks for your advice. It's done in c46755568f17909fbaab207a050155eedc6cec02 and also fixed a typo.